### ISSUE TITLE
Two different install methods in documentation

### ISSUE THREAD

**Issue by borpin** [ryansch/docker-unifi-rpi#38]
There are 2 different methods of installing the controller in the documentation;

https://github.com/ryansch/docker-unifi-rpi#on-rpi

and

https://github.com/ryansch/docker-unifi-rpi/wiki/Getting-Started-on-a-Raspberry-Pi#on-rpi

I used the first (thank you very much 😄) but I have no idea how I might upgrade it.  There is an instruction on upgrading under the second method.

**Comment by ryansch** [comment]
Yeah it seems like streamlining the documentation would really help. I've gotten some other feedback that points in that direction.  I'll leave this open until that work is completed.

**Comment by jftuga** [comment]
The second method worked well for me.  I was able to export a controller backup from my bare metal RPi and then start your docker image (method 2) on a different RPi.  The first time I visited the new controller website, it asked my if I want to restore from backup, which I did.  The restore process worked great.  My three APs are also now "connected" to the new controller using a different IP address.
